extras/ams_wrapper/src/api/types.py

```diff
@@ -46,6 +46,17 @@ def as_dict(self):
         }
         return result_dict
 
+class Attribute_fix:
```

Comment:
It would be better to avoid duplicated classes. The original class should be just updated or if something is shared with other model categories - more meaningful name should be used.